src/lib/jitter/info.txt

```diff
@@ -0,0 +1,19 @@
+# https://botan.randombit.net/handbook/dev_ref/configure.html#adding-a-new-module
+
+load_on never
+
+<module_info>
+name -> "Add jitter entropy to the system RNG"
+</module_info>
+
+<defines>
+JITTER -> 20240819
```

Comment:
This file should move to either `rng` or `entropy` (depending on how you want to present this) and the define updated accordingly